solutions/swb-reference/integration-tests/support/models/environments.ts

```diff
@@ -0,0 +1,12 @@
+export type EnvironmentStatus = 'STOPPED' | 'STOPPING' | 'FAILED' | 'COMPLETED' | 'PENDING' | 'TERMINATING';
```

Comment:
This already exist here
https://github.com/awslabs/solution-spark-on-aws/blob/b045c4c0d21724d13ae7e6e92c6dda000e6e7dc6/workbench-core/environments/src/constants/environmentStatus.ts

Reply:
replaced by type in `workbench-core`